### ISSUE TITLE
Clarify what is being shown after tapping Send

### ISSUE THREAD

**Issue by lognaturel** [opendatakit/share#50]
After tapping "Send," the list of forms is replaced by a list of filled forms. This is not very clear. Maybe the title could be something like "Filled forms to send" or something like that to provide the user with more context on where they are and what action they can now take.

**Comment by lognaturel** [comment]
It would also really help to know which filled forms were received vs. filled out on that device.

**Comment by moksh-mahajan** [comment]
Can I work on this isssue? @lakshyagupta21 @shobhitagarwal1612

**Comment by lakshyagupta21** [comment]
Sure, you can claim this issue by typing `@opendatakit-bot claim`

**Comment by moksh-mahajan** [comment]
To be clear, I just need to change the title of the tabs, right?

**Comment by lakshyagupta21** [comment]
Nope, on the main screen, there are two buttons send and receive so you need to change their text in such a way they give more context about their functionality.

**Comment by moksh-mahajan** [comment]
Okay!

**Comment by moksh-mahajan** [comment]
@opendatakit-bot claim

**Comment by moksh-mahajan** [comment]
@lakshyagupta21 @lakshyagupta21 Submitted a PR. Please review it.

**Comment by shobhitagarwal1612** [comment]
> It would also really help to know which filled forms were received vs. filled out on that device.

We're you able to add this info to the filled form list?

**Comment by moksh-mahajan** [comment]
@opendatakit-bot unclaim

**Comment by naman653** [comment]
@lakshyagupta21 @shobhitagarwal1612 I want to work on this issue. But I can't understand it quite much.

**Comment by shobhitagarwal1612** [comment]
The UX is not very intuitive right now. So what @lognaturel means is that the title of the opened activity should be improved in order to give clear intention of the opened activity

**Comment by devanshi7799** [comment]
@shobhitagarwal1612 @lakshyagupta21 i would like to work on this one. 
but still i am not clear about the issue. what is to be done.

**Comment by chidauri** [comment]
> It would also really help to know which filled forms were received vs. filled out on that device.

Would it be okay to change "Finalized on......" to reflect those forms which were received as "Received on......"

And for the original issue, i think in the title we can show "Send", since we now have separate tabs for FILLED FORMS and BLANK FORMS. wdyt?

**Comment by devanshi7799** [comment]
@opendatakit-bot claim
